eng/InternalTools.props

```diff
@@ -9,6 +9,8 @@
     </PropertyGroup>
     
     <ItemGroup>
+        <!-- Add explicit Newtonsoft.Json dependency to override the implicit 10.0.1 version brought in by Microsoft.DevDiv.Optimization.Data.PowerShell -->
+        <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion" Condition="'$(UsingToolVisualStudioIbcTraining)' == 'true'" />
```

Comment:
Looks like it would be high effort to get builds of the Optimization.Data.PowerShell with an updated Newtonsoft.Json dependency. Trying an explicit dependency here.

Reply:
@JoeRobich do we have a tracking bug to get this looked at?

Reply:
I opened a tracking issue to remove this reference once an updated build of the Optimization.Data.Powershell package is available https://github.com/dotnet/roslyn/issues/63450